packages/devtools/cli/package.json

```diff
@@ -31,16 +31,19 @@
   },
   "dependencies": {
     "@dxos/async": "workspace:*",
+    "@dxos/bare-template": "workspace:*",
```

Comment:
Did you consider `npm install`-ing the requested template as needed rather than including all the templates as cli deps?

The template name to cli could fit into a `@dxos/${template}-template` template string, then it could be installed globally or in a temp directory so it could be imported to execute.

This is probably simpler for now, but can't help but consider the possibility of support community templates like we've played with in previous iterations.

Reply:
oh yeah totally did consider it and opted to leave that for later yep.

Reply:
we can even `exec(`npx init @dxos`)` and not worry about this problem